Formula/soapysdr.rb

```diff
@@ -0,0 +1,29 @@
+class Soapysdr < Formula
+  desc "Vendor and platform neutral SDR support library"
+  homepage "https://github.com/pothosware/SoapySDR/wiki"
+  url "https://github.com/pothosware/SoapySDR/archive/soapy-sdr-0.8.0.tar.gz"
+  sha256 "cff3b1fb1cd3b88932f37df42f69fc3d8e0ea42f4788b35131ceaf5d1b908ac2"
+  license "BSL-1.0"
+  head "https://github.com/pothosware/SoapySDR.git"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "python@3.9"
+
+  def install
+    args = std_cmake_args
+    args << "-DENABLE_PYTHON=OFF"
+    args << "-DENABLE_PYTHON3=ON"
+    args << "-DSOAPY_SDR_ROOT=#{HOMEBREW_PREFIX}"
+    args << "-DSOAPY_SDR_EXTVER=release" unless build.head?
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "#{bin}/SoapySDRUtil", "--check=null"
```

Comment:
Is this a command a user might run when using this? Does this produce output we can check?

Reply:
There's many drivers a user can install but this module does not include any so checking for null is the only possible option.

```
SoapySDRUtil --check=null
######################################################
##     Soapy SDR -- the SDR abstraction library     ##
######################################################

Loading modules... done
Checking driver 'null'... PRESENT
```

Reply:
```SoapySDRUtil --info``` has an output like this if you prefer:

```
######################################################
##     Soapy SDR -- the SDR abstraction library     ##
######################################################

Lib Version: v0.8.0-release
API Version: v0.8.0
ABI Version: v0.8
Install root: /usr/local
Search path:  /usr/local/lib/SoapySDR/modules0.8
Module found: /usr/local/lib/SoapySDR/modules0.8/libaudioSupport.so  (0.1.1)
Module found: /usr/local/lib/SoapySDR/modules0.8/librtlsdrSupport.so (0.3.2)
Module found: /usr/local/lib/SoapySDR/modules0.8/libuhdSupport.so    (0.4.1)
Available factories... audio, rtlsdr, uhd
Available converters...
 -  CF32 -> [CF32, CS16, CS8, CU16, CU8]
 -  CS16 -> [CF32, CS16, CS8, CU16, CU8]
 -  CS32 -> [CS32]
 -   CS8 -> [CF32, CS16, CS8, CU16, CU8]
 -  CU16 -> [CF32, CS16, CS8]
 -   CU8 -> [CF32, CS16, CS8]
 -   F32 -> [F32, S16, S8, U16, U8]
 -   S16 -> [F32, S16, S8, U16, U8]
 -   S32 -> [S32]
 -    S8 -> [F32, S16, S8, U16, U8]
 -   U16 -> [F32, S16, S8]
 -    U8 -> [F32, S16, S8]
```

Reply:
Ok, let's do this:
```suggestion
    assert_match "Loading modules... done", shell_output("#{bin}/SoapySDRUtil --check=null")
```

Reply:
Done